layouts/partials/header.html

```diff
@@ -43,6 +43,11 @@
 		{{ $darkstyle := resources.Get "css/dark.css" | fingerprint }}
 		<link rel="stylesheet" type="text/css" href="{{ $darkstyle.Permalink }}" {{ if eq .Site.Params.mode "auto" }}media="(prefers-color-scheme: dark)"{{ end }} />
 	{{ end }}
+
+	{{- if or (eq .Site.Params.mode "auto") (eq .Site.Params.mode "light") -}}
+		{{ $darkstyle := resources.Get "css/dark.css" | fingerprint }}
+		<link disabled id="dark-mode-theme" rel="stylesheet" href="{{ $darkstyle.Permalink }}">
```

Comment:
@athul this is where, it critical. if theme is not dark mode, we can load the dark.css, but if theme are dark should we load main.css? It sound weird to me